os/include/tinyara/serial/tioctl.h

```diff
@@ -202,6 +202,7 @@
 #define TIOCCLRSIG      _TIOC(0x002f)	/* Clear signal to handle Interrupt */
 #define TIOCTXSTATE     _TIOC(0x0030)	/* Get State of TX */
 #define TIOCRXSTATE     _TIOC(0x0031)	/* Get State of RX */
+#define TIOCM_LOOP	_TIOC(0x0032)	/* Enable/Disable loopback mode */
```

Comment:
How about keep the naming policy?
```
TIOCLOOPBACK
```

Reply:
Why we support the driver is keeping the same app code regardless of BSP.
To guarantee above, we should guide which value is loopback on and which value is off.
Could you add the definitions of LOOPBACK_ON and OFF?

Reply:
i changed the name because taejun suggested that:

`For most of constant value, we usually follow linux's, there's TIOCM_LOOP in linux. Please check it.`

now, let me know what you think?